src/components/reusable/drawer/caregiver-drawer/CaregiverDrawer.tsx

```diff
@@ -0,0 +1,195 @@
+import { TabContext } from '@mui/lab';
+import { Box, IconButton, Stack } from '@mui/material';
+import { SyntheticEvent, useState } from 'react';
+import ArrowBackFilled from 'src/assets/icons/ArrowBackFilled';
+import FreeCancellation from 'src/assets/icons/FreeCancellation';
+import MonetizationOn from 'src/assets/icons/MonetizationOn';
+
+import Drawer from 'src/components/reusable/drawer/Drawer';
+import {
+  DrawerBody,
+  DrawerFooter,
+  DrawerHeader,
+  DrawerTitle,
+} from 'src/components/reusable/drawer/styles';
+import { useLocales } from 'src/locales';
+import { appointmentApi } from 'src/redux/api/appointmentApi';
+
+import { getMockCaregiverAvatar } from 'src/components/create-appointment-fourth/helpers';
+import { BIG_CAREGIVER_AVATAR_SIZE, FIRST_SELECTED_TAB } from './constants';
+import { formatWorkExperienceDateRange, formatWorkExperienceDateRangeTenure } from './helpers';
+import {
+  DrawerAvatar,
+  DrawerDateTenure,
+  DrawerItem,
+  DrawerStats,
+  DrawerTextHeading,
+  DrawerTextTitle,
+  DrawerTextValue,
+  StyledLink,
+  StyledTab,
+  StyledTabPanel,
+  StyledTabs,
+  StyledUnorderedList,
+  StyledVideo,
+} from './styles';
+
+interface CaregiverDrawerProps {
+  open: boolean;
+  onClose: () => void;
+  caregiverId: string;
+  footer?: React.ReactNode;
+}
+
+export default function CaregiverDrawer({
+  open,
+  onClose,
+  caregiverId,
+  footer,
+}: CaregiverDrawerProps): JSX.Element | null {
+  const { translate, currentLang } = useLocales();
+
+  const [selectedTab, setSelectedTab] = useState<string>(FIRST_SELECTED_TAB);
+
+  const { data: selectedCaregiver, isLoading } =
+    appointmentApi.useGetCaregiverDetailsQuery(caregiverId);
+
+  if (isLoading) return null;
+  if (!selectedCaregiver) return null;
+
+  const handleTabClick = (_event: SyntheticEvent<Element, Event>, newValue: string): void => {
+    setSelectedTab(newValue);
+  };
+
+  return (
+    <Drawer open={open} onClose={onClose}>
+      <DrawerHeader>
+        <IconButton size="small" onClick={onClose}>
+          <ArrowBackFilled />
+        </IconButton>
+        <DrawerTitle>
+          {selectedCaregiver.firstName} {selectedCaregiver.lastName}
+        </DrawerTitle>
+      </DrawerHeader>
+      <DrawerStats flexDirection="row">
+        <DrawerAvatar
+          src={getMockCaregiverAvatar(BIG_CAREGIVER_AVATAR_SIZE)}
+          alt={`${selectedCaregiver.firstName} ${selectedCaregiver.lastName}`}
+        />
+        <Stack alignItems="center" flexDirection="row" justifyContent="space-evenly" width="100%">
+          <Stack flexDirection="column" alignItems="center">
```

Comment:
It's not really inline styles, maybe ok if 1-2, but here just to many. Would be a better practise to create styled component

Reply:
fixed